### ISSUE TITLE
Store data created run-time in R

### ISSUE THREAD

**Issue by laurivesa** [openforis/arena#1304]
We need to get space (and access to that space) where to store CSVs and other files created during the running the R scripts.

**Comment by minotogna** [comment]
can't the user download the files before closing the browser ?

**Comment by minotogna** [comment]
the result of user generated files is unpredictable, therefore it is very difficult to provide such solutions that is efficient and robust

**Comment by laurivesa** [comment]
yes, but there should be a system so that user can notice to do this easily

**Comment by minotogna** [comment]
wouldn't this be enough https://support.rstudio.com/hc/en-us/articles/200713893-Uploading-and-Downloading-Files :

> Downloading Files
> 
> To download files from RStudio Server you should take the following steps:
> 
> Switch to directory you want to download files from within the Files pane
> Select the file(s) and/or folder(s) you want to download
> Click More -> Export on the toolbar
> You'll then be prompted with a default file name for the download. Either accept the default or specify a custom name then press OK.
> Note that if you select multiple files or folders for download then RStudio compresses all of the files into a single zip archive for downloading.
>

**Comment by laurivesa** [comment]
Hi Mino,
This method is possible when developer uses RStudio. But the end users
should run without starting Rstudio, and they should be able to download
the output files creaed by the script. Can be add ' download.file' methods
(
https://www.rdocumentation.org/packages/utils/versions/3.6.2/topics/download.file
)
into the scripts so that downloading is automatically started?

cheers, Lauri
